### DISCUSSION TITLE
Custom voice servers?

### DISCUSSION THREAD

**Discussion by awildboop** [benawad/dogehouse#2299] [Q&A]
Out of curiosity, would it be possible to allow people to actually host their own voice servers on their own hardware in the future?

**Comment by theotterlord** [comment]
You can host dogehouse locally (or on a server), but it will obviously be completely seperate from the real thing. I don't think there are any plans for this, and I'm not sure I see a benefit 🤔

**Reply by awildboop** [reply]
Well, it'd be nice to be able to host separate servers to reduce load on a central voice server - plus, it'd be interesting to try and make a more performant custom server using a faster language (such as C, Go, or Rust).
